api/prisma/seed-staging.ts

```diff
@@ -103,6 +103,7 @@ export const stagingSeed = async (
         FeatureFlagEnum.enablePartnerDemographics,
         FeatureFlagEnum.enablePartnerSettings,
         FeatureFlagEnum.enableListingsPagination,
+        FeatureFlagEnum.disableCommonApplication,
```

Comment:
chore: instead of setting this on the "Bridge Bay" jurisdiction, can you set it on "Lakeview". 

Lakeview is supposed to roughly mimic Detroit and Bridge Bay mimics HBA